packages/web/src/common/store/pages/signon/reducer.js

```diff
@@ -224,6 +227,24 @@ const actionsMap = {
       verified: action.profile.is_verified
     }
   },
+  [SET_TIKTOK_PROFILE](state, action) {
+    return {
+      ...state,
+      tikTokProfile: action.tikTokProfile,
+      name: {
+        value: action.profile.display_name || '',
```

Comment:
Maybe destructure action

Reply:
I'm gonna leave this for now bc it's copy and paste from the other ones and I don't want to muck with a non-ts file